documentation/source/user_section/tutorials/registration-to-template/rootlets-based-registration/applying-the-registration-algorithm.rst

```diff
@@ -0,0 +1,31 @@
+.. _applying-registration-rootlets-algorithm:
+
+Applying the registration algorithm
+###################################
+
+To apply the registration algorithm, the following command is used:
+
+.. code:: sh
+
+   sct_register_to_template -i t2.nii.gz -s t2_seg.nii.gz -lrootlet t2_rootlets.nii.gz -c t2 -ofolder rootlets-reg -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : Input image
+   - ``-s`` : Segmented spinal cord corresponding to the input image
+   - ``-lrootlet`` : Dorsal and ventral nerve rootlets segmentation. Example: `anat_rootlets.nii.gz`
+   - ``-c`` : Contrast of the image. Specifying this determines the version of the template to use. (Here, ``-c t2`` means that ``PAM50_t2.nii.gz`` will be used.)
+   - ``-ofolder`` : Output folder where the results will be saved. If the folder does not exist, it will be created.
+   - ``-qc`` : Directory for Quality Control reporting. QC reports allow us to evaluate the results slice-by-slice.
+
+:Output files/folders:
+   - ``anat2template.nii.gz`` : The anatomical subject image (in this case, ``t2.nii.gz``) warped to the template space.
+   - ``template2anat.nii.gz`` : The template image (in this case, ``PAM50_t2.nii.gz``) warped to the anatomical subject space.
+   - ``warp_anat2template.nii.gz`` : The 4D warping field that defines the transform from the anatomical image to the template image.
+   - ``warp_template2anat.nii.gz`` : The 4D warping field that defines the inverse transform from the template image to the anatomical image.
+
+Once the command has finished, at the bottom of your terminal there will be instructions for inspecting the results using :ref:`Quality Control (QC) <qc>` reports. Optionally, If you have :ref:`fsleyes-instructions` installed, a ``fsleyes`` command will printed as well.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/sb/add-fig-rootlets-reg/rootlets-based-registration/output-registration-rootlets.png
+   :align: center
+
+   Input/output images for :ref:`sct_register_to_template`
```

Comment:
Now that the PR has been merged and the branch deleted, these images need to be updated to point to `master` I think:

```suggestion
.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/rootlets-based-registration/output-registration-rootlets.png
   :align: center

   Input/output images for :ref:`sct_register_to_template`
```